sites/example-project/src/components/ui/Formatting/BuiltInFormatGrid.svelte

```diff
@@ -1,5 +1,4 @@
 <script>
-  import "./format-grid.css";
   import { defaultExample, formatExample } from "$lib/modules/formatting";
```

Comment:
I assumed importing css would result in svelte localizing the classes to the importing component.  Doesn't seem to work that way. I think copying is fine for now but we should figure out the correct way to do it at some point.  I suspect this won't work either  `<style src='./format-grid.css'>`